packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/nodejs/githubinstructions.md

```diff
@@ -14,3 +13,10 @@ The sample applications require the installation of the module dependencies list
 1. Navigate to a sample application root folder. For example, `okta-auth-js/samples/generated/express-embedded-sign-in-widget`.
 
 2. Install the dependencies: `npm install`.
+
+The source code from the sample app and SDK are located here: `.../okta-auth-js/`. The project structure consists of:
```

Comment:
The project structure information is in a different place to where it is in the other platform instructions. Any reason for that? I think it would work in either position, but I'm just thinking of consistency.